addons/mail/static/tests/discuss/core/channel_invitation.test.js

```diff
@@ -154,14 +158,13 @@ test("unnamed group chat should display correct name just after being invited",
             channel_type: "group",
         },
     ]);
-    const { env } = await start();
+    const env = await start();
     await openDiscuss();
     await contains(".o-mail-DiscussSidebarChannel", { text: "General" });
     await contains(".o-mail-DiscussSidebarChannel", { count: 0, text: "Jane and Mitchell Admin" });
-    const currentPartnerId = serverState.partnerId;
```

Comment:
This change is possibly not ok. It's stored in a separate variable to avoid interference with `withUser` that could(should) change the current partner (if it's coded correctly)